spec/rspec/rails/matchers/be_a_new_spec.rb

```diff
@@ -147,13 +147,13 @@ def attributes
           end
 
           def new_record?; false; end
-        end.new(:foo => 'foo', :bar => 'bar')
+        end.new(foo: 'foo', bar: 'bar')
       end
 
       context "all attributes same" do
         it "fails" do
           expect {
-            expect(record).to be_a_new(String).with(:foo => 'foo', :bar => 'bar')
+            expect(record).to be_a_new(String).with(foo: 'foo', bar: 'bar')
```

Comment:
I'm afraid that with Ruby 2.7 we'll have to rework this back to hash rocket.
No worries, let's keep as is until this strikes us.